lib/src/types.ts

```diff
@@ -39,6 +39,7 @@ export type Schema = {
 export type Options = {
   schema?: Schema;
   resolver?: (blok: SbBlok) => ComponentNode;
+  stringResolver?: (str: string) => ComponentNode;
```

Comment:
I would rather call this function a `contentResolver` as it wraps content with additional logic.

Reply:
After all, we might call the same as react alternative has https://www.npmjs.com/package/storyblok-rich-text-react-renderer#text-resolver which is `textResolver` as it indeed takes the `text` node wrapping:
```
textResolver?: (text?: string) => ComponentNode;
```

Reply:
After all, we might call the same as react alternative has https://www.npmjs.com/package/storyblok-rich-text-react-renderer#text-resolver which is `textResolver` as it indeed takes the `text` node wrapping:
```
textResolver?: (text?: string) => ComponentNode;
```

Reply:
renamed it to textResolver

Reply:
alright, I remembered why I named it `stringResolver`: in the react library there are two functions `defaultStringResolver` and `textResolver`. `textResolver` is to resolve **text nodes**, while `defaultStringResolver` is to resolve plain strings https://github.com/claus/storyblok-rich-text-react-renderer/blob/master/src/index.js#L104C14-L104C14. Documentation there is a bit misleading

Reply:
Sure, its all fine, this API does not necessarily need to match 1:1 with Claus one. I am totally fine with having `textResolver` for, well, resolving text's (which basically the same as `defaultStringResolver` in react), and leaving existing availability for users to affect nodes via `schema.nodes.text` function.